.github/workflows/release-please.yml

```diff
@@ -0,0 +1,18 @@
+name: Run Release Please
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
```

Comment:
Should the release flow be included here?

Reply:
No, the release flow happens when the release tag is created whereas this flow occurs every time something new is pushed to main